charts/kube-prometheus-stack/templates/grafana/configmaps-datasources.yaml

```diff
@@ -16,8 +16,8 @@ data:
   datasource.yaml: |-
     apiVersion: 1
     datasources:
-{{- if .Values.grafana.sidecar.datasources.defaultDatasourceEnabled }}
-    - name: Prometheus
+{{- if .Values.grafana.sidecar.datasources.defaultDatasource.enabled }}
```

Comment:
This makes sense but bump the major version and describe the changes in the readme